src/Data/Yaml.hs

```diff
@@ -69,12 +69,13 @@ module Data.Yaml
     , ToJSON (..)
     , FromJSON (..)
       -- * Custom encoding
+    , objToEvents
```

Comment:
I think I'd rather see this exported from `Data.Yaml.Internal`. Also, please ensure:

* Newly exported identifiers have a Haddock including a `@since` comment
* Changes are reflected in the ChangeLog
* A minor version bump is added to the cabal file

Reply:
👍

Reply:
It looks like moving `objToEvents` to `Data.Yaml.Internal` would require moving the `EncodeOptions` type in there as well (since it uses the constructors), which would require moving there pretty much everything else as well (for the same reason)?